.github/CODEOWNERS

```diff
@@ -7,6 +7,3 @@
 /client/ @thefallentree @xli
 # Subscribe to LCS format changes
 /testsuite/generate-format/tests/staged/libra.yaml @thefallentree @matbd @xli
-# Subscribe to TCB dependency summary changes
-/summaries/summary-lsr.toml @davidiw
```

Comment:
oh, this explains a lot :P

I agree that without /requiring/ a review from a codeowner this can appear to be noise :/